app/src/main/java/org/mozilla/focus/home/FeatureSurveyViewHelper.kt

```diff
@@ -88,7 +89,10 @@ class FeatureSurveyViewHelper internal constructor(private val context: Context,
                     // Open Play Store in a new tab
                     val url = AppConfigWrapper.getVpnRecommenderUrl(context)
                     if (!TextUtils.isEmpty(url)) {
-                        ScreenNavigator.get(context).showBrowserScreen(url, true, false)
+                        // fallback to open play store link using WebView
```

Comment:
Maybe: // Force to load with GooglePlay instead of Webview to get parameters correctly passed to Google Play. The "fallback..." seems to be weird here to me.